addon/utils/text.js

```diff
@@ -71,8 +72,9 @@ export function trimDataToFit (text, width, font) {
  * Note: expects data-text attribute to be on element with full text
  *
  * @param {HTMLElement} element - HTML element to trim data within
+ * @param {jQuery} [$] - jQuery selector, can be stubbed for testing
```

Comment:
I'm not sure I get what you're doing here as the tests appear to stub Ember.$ not the global jQuery.